controllers/email/email.js

```diff
@@ -0,0 +1,42 @@
+const nodemailer = require("nodemailer");
+const sgMail = require("@sendgrid/mail");
+const ejs = require("ejs");
+const path = require("path");
+require("dotenv").config();
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const transporter = nodemailer.createTransport({
+    service: "SendGrid",
+    auth: {
+        user: "apikey",
+        pass: process.env.SENDGRID_API_KEY,
+    },
+});
+
+module.exports = {
+    sendVerificationEmail: async (email, verificationToken, req) => {
+        try {
+            const verificationLink = `${req.protocol}://${req.get(
+                "host"
+            )}/api/users/verify/${verificationToken}`;
+
+            const html = await ejs.renderFile(
```

Comment:
Nie znam tej paczki, właśnie ja sprawdziłem i wydaje się że to nie jest zbyt duże narzędzie do tak małego zadania.